tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py38
+envlist = py38
```

Comment:
@aibrahim-cw @hoolymama tox steps were removed from CI/CD in this PR, can we delete this unused file?